optuna_dashboard/ts/apiClient.ts

```diff
@@ -137,6 +138,9 @@ export const getStudyDetailAPI = (
         objective_names: res.data.objective_names,
         form_widgets: res.data.form_widgets,
         is_preferential: res.data.is_preferential,
+        feedback_component_type: res.data.feedback_component_type ?? {
+          output_type: "note",
+        },
```

Comment:
I think you should not insert the default value on `apiClient`. How about inserting a default value on the JSON API layer or making this field allows undefined?